src/api/save-idea/content-types/save-idea/schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "kind": "collectionType",
+  "collectionName": "save_ideas",
+  "info": {
+    "singularName": "save-idea",
+    "pluralName": "save-ideas",
+    "displayName": "SaveIdea",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "ObjectId": {
+      "type": "uid"
+    },
+    "users": {
+      "type": "relation",
+      "relation": "manyToMany",
+      "target": "plugin::users-permissions.user",
+      "inversedBy": "save_ideas"
+    },
+    "SavedAt": {
+      "type": "datetime"
+    },
+    "idea_card": {
```

Comment:
if we already have the `ObjectId`, is this necessary?

Reply:
The updates have been implemented.